packages/@ember/application/tests/dependency_injection/default_resolver_test.js

```diff
@@ -198,6 +198,7 @@ moduleFor(
     }
 
     [`@test no assertion for routes that extend from Route`](assert) {
+      assert.test.assertions = []; // clear assertions that occurred in beforeEach
```

Comment:
Hmm, why do we need to clear the assertions?

Reply:
In beforeEach we are creaing an application using the default resolver.